examples/dcosattributes/dcos1.8.4.attributes.json

```diff
@@ -0,0 +1,45 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "DCOS184"

```

Comment:
Two things in examples:
1. change this to "DCOS" only
2. add a README.md to this folder that describes the attributes for DCOS.  Only has to be a few sentences (what it does, and how to specify a container within DCOS).  An example README.md is under vnet/ example folder.